mixer/template/template.gen.go

```diff
@@ -3896,6 +3986,46 @@ type builder_tracespan_Template struct {
 	// builder for field httpStatusCode: int64.
 
 	bldHttpStatusCode compiled.Expression
+
+	// builder for field client_span: bool.
+
+	bldClientSpan compiled.Expression
```

Comment:
Would it be better to represent the span kind here, rather than a boolean, as the span kind can be "client", "server" or internal (no span kind defined) - and even "producer" or "consumer" if describing a messaging exchange (not immediately relevant but possibly in the future).